brainstormforce/ultimate-addons-for-gutenberg
# Fix: Responsive condition - Deactivate all the modules Responsive Conditions keep is active.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/2634da90-a8d4-4a2c-9d29-a36878a5130f?redirect=true&view=06c01065-22db-4b44-9a83-f0eceb06e66a&t=attributes

### Screenshots

- https://share.getcloudapp.com/p9u6DrNb

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Go to Spectra setting.
- Click on Blocks, Try deactivating all and activating all.
- Responsive condition should be deactivated when Deactivating all.
- Responsive condition should be activated when Activating all.

### Checklist:
